onnxscript/values.py

```diff
@@ -207,7 +207,7 @@ def adapt(input: ExtendedModeValue) -> EagerModeValue:
         elif input is None:
             return None
         elif isinstance(input, list):
-            return [adapt(elt) for elt in input]
+            return input
```

Comment:
I am still hesitant about this. What happens for example when we call Concat with a list of numpy arrays?

Reply:
> ~I am still hesitant about this. What happens for example when we call ConcatSequence with a list of numpy arrays?~
> 
> nvm. Looks like it works. does this look good to you @gramalingam ?

I just found out the issue. Actually, this change is not necessary, we should reverse it back.

Below is the code which calls this function:

> def __call__(self, *args, **kwargs):
        """Implements an eager-mode execution of an onnxscript function."""
        new_args, has_array = _adapt_to_eager_mode(args)
        result = self.function(*new_args, **kwargs)

Ideally, only inputs will be passed into _adapt_to_eager_mode for this adaption, those attributes are not supposed to be passed into this method because attributes should be in kwargs.

The real problem is somehow, both inputs and attributes were saved into args hence they were passed into _adapt_to_eager_mode() together. We should remove those attributes from args so that they won't be passed into _adapt_to_eager_mode() method.

Reply:
I agree with @fatcat-z : the issue may be higher up, in distinguishing inputs from attributes. Fixing that may be the right fix.